fuzzing_docker.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
```

Comment:
This script should be in `bin` folder with other scripts so we don't clutter top folder? Dockerfile also? Or should we have a fuzzing folder for these? @jim-wang-intel , @bnevis-i , thoughts?

Reply:
yea, i like the idea to have a fuzzing folder and put all fuzz test related in that folder